### PR TITLE
Update headings to match sidebar

### LINKED ISSUE #2150: User Guide: Tweak heading to match site navigation menu
For example, consider the following heading:
image

Perhaps we can use `Using Components: Pop-Ups` instead, so that the two match?

Not a major issue. Just a cosmetic thing. 

Applies to some other pages as well.

### PR DESCRIPTION
**Pull request by itsyme**
**What is the purpose of this pull request?**

- [x] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

**Overview of changes:**
Resolves #2150. All headers which did not match the title in the sidebar were changed for both the User Guide and Developer Guide.

**Anything you'd like to highlight/discuss:**

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
Update headings to match sidebar

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No unrelated changes

### CODE REVIEW

File: docs/userGuide/components/popups.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 <include src="advanced.md#slots-info" />
 
-# Pop-Up Components
+# Using Components: Pop-ups
```
Comment (yucheng11122017) [inline docs/userGuide/components/popups.md:17]: Small nit: Should be Pop-Ups according to side bar

**Review by yucheng11122017 — COMMENTED** [review]
Thanks for the work @itsyme!

For user guide, the .gitignore file and glossary doesn't match as well. 
For the dev guide, the introduction (under onboarding bootcamp) doesn't match. 
DG the subpages under the github actions are titled "GitHub Actions: {{title}}". Same for subpage in Appendices. Not sure if this should be changed?

**Comment by itsyme** [comment]
Thanks for the detailed review @yucheng11122017 !! I have made the relevant changes in the latest commit. For Onboarding Bootcamp -> Introduction, . As for "GitHub Actions: `{{ title }}`", the correct titles were set at the top of the page such that `{{ title }}` could be reused in other places in the page as well.  I agree that it might be good to standardise the use of `{{ title }}` across all pages in the future.

**Comment by yucheng11122017** [comment]
> Thanks for the detailed review @yucheng11122017 !! I have made the relevant changes in the latest commit. For Onboarding Bootcamp -> Introduction, . As for "GitHub Actions: `{{ title }}`", the correct titles were set at the top of the page such that `{{ title }}` could be reused in other places in the page as well. I agree that it might be good to standardise the use of `{{ title }}` across all pages in the future.

Yup agree! It seems like only sub headings (like under components) have the {{ heading }}: {{ title }} format while the rest all use {{ title }}

**Comment by yucheng11122017** [comment]
> > Thanks for the detailed review @yucheng11122017 !! I have made the relevant changes in the latest commit. For Onboarding Bootcamp -> Introduction, . As for "GitHub Actions: `{{ title }}`", the correct titles were set at the top of the page such that `{{ title }}` could be reused in other places in the page as well. I agree that it might be good to standardise the use of `{{ title }}` across all pages in the future.
> 
> Yup agree! It seems like only sub headings (like under components) have the {{ heading }}: {{ title }} format while the rest all use {{ title }}

Maybe this would be changed to {{ title }} to follow the rest

**Comment by itsyme** [comment]
Hi @yucheng11122017! I think that's a great suggestion! However, I think this may be out of scope for this PR. Perhaps it could be good to open a new issue/PR to address this issue!

**Comment by jovyntls** [comment]
What do you think of not having the `Using Components:` prefix on the component titles? I left a [comment](https://github.com/MarkBind/markbind/issues/2150#issuecomment-1427003310) on the original issue about this - personally I'd prefer to not have the prefix since it's a bit verbose and this naming pattern is more suited for breadcrumbs than for a page title 😅

**Comment by yucheng11122017** [comment]
> What do you think of not having the `Using Components:` prefix on the component titles? I left a [comment](https://github.com/MarkBind/markbind/issues/2150#issuecomment-1427003310) on the original issue about this - personally I'd prefer to not have the prefix since it's a bit verbose and this naming pattern is more suited for breadcrumbs than for a page title 😅

Yeah I agree. Especially if there are sub sub sub.. headings it'll make the title really unnecessary and long

**Comment by itsyme** [comment]
Alright! I think that is cleaner as well. I will update it in the next commit!

**Comment by itsyme** [comment]
Hi @yucheng11122017! I realised I have not resolved your comment about developer guide -> onboarding bootcamp -> introduction. I left the title as "Onboarding Bootcamp" as I felt that putting "Introduction" alone for the title might be a little too vague. What do you think?

**Comment by yucheng11122017** [comment]
> Hi @yucheng11122017! I realised I have not resolved your comment about developer guide -> onboarding bootcamp -> introduction. I left the title as "Onboarding Bootcamp" as I felt that putting "Introduction" alone for the title might be a little too vague. What do you think?

Yeah good point! Concur with you on that then

**Comment by jovyntls** [comment]
One last nit since we're standardising titles, can we make the capitalisation here consistent? ('sites'->'Sites')
image

**Review by jovyntls — APPROVED** [review]
LGTM

**Pull request merged**